src/config/loading.rs

```diff
@@ -5,15 +5,14 @@ use std::{
     sync::Mutex,
 };
 
-use glob::glob;
-use indexmap::IndexMap;
-use lazy_static::lazy_static;
-
 use super::{
     builder::ConfigBuilder, format, validation, vars, ComponentKey, Config, ConfigPath, Format,
     FormatHint, TransformOuter,
 };
 use crate::signal;
+use glob::glob;
+use indexmap::IndexMap;
+use once_cell::sync::Lazy;
```

Comment:
Seems like some unintended changes got pulled in. I've seen this in other PRs as well, I wonder if it's because of a manually ran [`rustfmt` configuration](https://github.com/vectordotdev/vector/blob/f049246db7e3025aa55eb1e160bd3669b485887d/.rustfmt.toml#L4) (via #10496) doesn't match people's editor configs.

Reply:
Yeah that seems likely. Might be worth having people take a look at their local configs and try to align them with what we did there.